README.md

```diff
@@ -129,7 +129,44 @@ This is Day to Day Work Linux Cheatsheet for Software Engineers.
 |Undo| u
 |Redo| ctrl + r
 
+## Bash Commands
+| Name   |   Command |
+|------------ | -------------|
+|Show System| uname -a
+|Show Mounted File System| mount
+|Show System Date| date
+|Show Uptime| uptime
+|Show username| whoami
+|Show Manual Command| man `<command>`
+|Show History Command| history
+
+## Bash Shortcuts
+| Name   |   Command |
+|------------ | -------------|
+|Stop Current Command| CTRL + c
+|Sleep Program| CTRL + z
+|Search History| CTRL + r
+|Repeat Last Command| !!
+|Run Last Command Start with Given String| !`<string>`
```

Comment:
this line is mistake, please fix this line

Reply:
Ok I already fix it. Is the current one is True ?

Reply:
These should be like this !

|Sleep Program| CTRL + z
|Search History| CTRL + r
|Repeat Last Command| !!
|Run Last Command Start with Given String| !!`<string>`